### ISSUE TITLE
C89 "1.21.x" (Nightly) manual pass/regression check

### ISSUE THREAD

**Issue by kjozwiak** [brave/qa-resources#207]
#### Release Date/Target:

* Due Date: **`ASAP`** (needs to be completed before we can merge **`C89`** into `1.21.x` & `1.20.x`)

#### Summary:

QA needs to run through **`CR89`** on Nightly before it's uplifted into`1.21.x` & `1.20.x`. 

**`Note:`** Sync will need to be thoroughly checked/QA'd as that's where a lot of the changes are occurring and where the highest risk of possible regressions due to the **`C89`** changes.

#### Manual Passes/Documentation:

* Manual test run on Android for 1.21.x - Nightly (Checking C89) #13968 - https://github.com/brave/brave-browser/issues/13968

**Comment by kjozwiak** [comment]
@bbondy @mkarolin @jsecretan please let us know when everything landed into `1.21.x` and QA can begin going through https://github.com/brave/brave-browser/issues/13968.

**Comment by kjozwiak** [comment]
Closing as @GeetaSarvadnya completed the above 👍 

```
Geeta  8:17 AM
Testing on CR 89 is completed on Windows, the issues found while testing CR 89 are listed below. I didn't see any major settings change w.r.t CR89 (just one option added Card nickname in Payment methods)

* https://github.com/brave/brave-browser/issues/14101 - Regression - closed
* https://github.com/brave/brave-browser/issues/14065 - Regression - closed
* https://github.com/brave/brave-browser/issues/14062 - Regression - closed
* https://github.com/brave/brave-browser/issues/14130 - Open (It's a new issue not related CR89)
* https://github.com/brave/brave-browser/issues/14175 - Open (It's a new issue not related CR89)
```

@bsclifton will address/fix https://github.com/brave/brave-browser/issues/14130 as per the discussion during the b-c planning/triage meeting. https://github.com/brave/brave-browser/issues/14175 will be fixed some other time and doesn't need to get fixed before C89 is uplifted into `1.21.x` & `1.20.x`.
